{{cookiecutter.__src_folder_name}}/pyproject.toml

```diff
@@ -13,6 +13,7 @@ known-first-party = ["fastapi_app"]
 
 [tool.black]
 line-length = 120
+extend-exclude = "migrations"
```

Comment:
Did that work? I have a more complex syntax for extend-exclude, but maybe mine was overly complex? I can't remember now why I thought I couldn't use the simpler syntax.

https://github.com/search?q=owner%3Apamelafox%20extend-exclude&type=code

Reply:
Discussion: This will ignore *anything* with migrations in it. Probably okay for this repo but we could use the more specific syntax in future.

Reply:
I agree and definitely would support a more granular listing on this if we know exactly how these files would be passed in.